examples/regression/4-tutorials/plot_ts-tutorial.py

```diff
@@ -292,34 +425,68 @@ class that block bootstraps the training set.
 # Let's now compare the coverages obtained by MAPIE with and without update
 # of the residuals on a 24-hour rolling window of prediction intervals.
 
+rolling_coverage_aci_pfit, rolling_coverage_aci_npfit = [], []
+rolling_coverage_enbpi_pfit, rolling_coverage_enbpi_npfit = [], []
+
 window = 24
-rolling_coverage_pfit, rolling_coverage_npfit = [], []
+
 for i in range(window, len(y_test), 1):
-    rolling_coverage_pfit.append(
+    rolling_coverage_aci_npfit.append(
         regression_coverage_score(
-            y_test[i-window:i], y_pis_pfit[i-window:i, 0, 0],
-            y_pis_pfit[i-window:i, 1, 0]
+            y_test[i-window:i], y_pis_aci_npfit[i-window:i, 0, 0],
+            y_pis_aci_npfit[i-window:i, 1, 0]
         )
     )
-    rolling_coverage_npfit.append(
+    rolling_coverage_aci_pfit.append(
         regression_coverage_score(
-            y_test[i-window:i], y_pis_npfit[i-window:i, 0, 0],
-            y_pis_npfit[i-window:i, 1, 0]
+            y_test[i-window:i], y_pis_aci_pfit[i-window:i, 0, 0],
+            y_pis_aci_pfit[i-window:i, 1, 0]
+        )
+    )
+
+    rolling_coverage_enbpi_npfit.append(
+        regression_coverage_score(
+            y_test[i-window:i], y_pis_enbpi_npfit[i-window:i, 0, 0],
+            y_pis_enbpi_npfit[i-window:i, 1, 0]
+        )
+    )
+    rolling_coverage_enbpi_pfit.append(
+        regression_coverage_score(
+            y_test[i-window:i], y_pis_enbpi_pfit[i-window:i, 0, 0],
+            y_pis_enbpi_pfit[i-window:i, 1, 0]
         )
     )
 
 plt.figure(figsize=(10, 5))
 plt.ylabel(f"Rolling coverage [{window} hours]")
+
+plt.plot(
+    y_test[window:].index,
+    rolling_coverage_aci_npfit,
+    label="ACI Without update of residuals (NPfit)",
+    linestyle='--',
+)
 plt.plot(
     y_test[window:].index,
-    rolling_coverage_npfit,
-    label="Without update of residuals"
+    rolling_coverage_aci_pfit,
+    label="ACI With update of residuals (Pfit)",
+    linestyle=':',
 )
+
 plt.plot(
     y_test[window:].index,
-    rolling_coverage_pfit,
-    label="With update of residuals"
+    rolling_coverage_enbpi_npfit,
+    label="ENBPI Without update of residuals (NPfit)",
+    linestyle='-.',
 )
+plt.plot(
```

Comment:
We can't really see all the lines, the blue line is difficult to see.

Reply:
Done @LacombeLouis :)